recipes/lora_finetune_single_device.py

```diff
@@ -263,11 +267,9 @@ def _setup_model(
         if compile_model:
             log.info("Compiling model with torch.compile...")
             model = utils.wrap_compile(model)
-        log.info(
-            utils.memory_stats_log(
-                "Memory Stats after model init:", device=self._device
-            )
-        )
+        if self._device == torch.device("cuda"):
```

Comment:
This is for the CPU recipe tests?

Reply:
yes, this actually wont throw the log just doesn't print anything